docs/docker-quickstart.md

```diff
@@ -39,34 +33,23 @@ Second, choose a tag for your build. The tag will be added
 to the image as the contents of the file
 `/opt/hedera/services/.VERSION`. A reasonable tag is the output of
 `git describe --tags --always --dirty`; for example,
-`"oa-release-r5-rc6-13-gf18d2ff77-dirty"`. Ensure the
-Docker Compose [.env file](../.env) has an empty registry prefix
-and your tag:
+`"oa-release-r5-rc6-13-gf18d2ff77-dirty"`. 
+Ensure the Docker Compose .env file `(../docker/.env)` has an empty registry prefix and your tag:
 
 ```
 TAG=oa-release-r5-rc6-13-gf18d2ff77-dirty
 REGISTRY_PREFIX=
 ```
 
-Third, build the image with an empty registry prefix and the `TAG` from your `.env` file:
+The file can be created by calling the `updateDockerEnv` gradle task (execute `./gradlew updateDockerEnv` from the root folder of the repo).
```

Comment:
I prefer to have commands for the terminal in a separate line (within a code block).